Library/Homebrew/utils.rb

```diff
@@ -467,6 +467,13 @@ def paths
     end.uniq.compact
   end
 
+  def parse_author!(author)
+    /^(?<name>[^<]+?)[ \t]*<(?<email>[^>]+?)>$/ =~ author
+    raise "Unable to parse name and email." if name.blank? && email.blank?
```

Comment:
```suggestion
    raise UsageError, "Unable to parse name and email." if name.blank? && email.blank?
```

If you want `--help` output printed. Otherwise fine as-is.

Reply:
There's another (somewhat) related issue I noticed here – using `<` and `>` underlines the enclosed content in the manpages, but I can't seem to figure out how to prevent that (and actually print the angular brackets). Any suggestions?